ztp/ztp-policy-generator/README.md

```diff
@@ -0,0 +1,318 @@
+    name: site-du-sno-1-sriov-nnp-mh-policy
+    namespace: sites-sub
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+spec:
+    remediationAction: enforce
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: site-du-sno-1-sriov-nnp-mh-policy-config
+            spec:
+                remediationAction: enforce
+                severity: low
+                namespaceselector:
+                    exclude:
+                        - kube-*
+                    include:
+                        - '*'
+                object-templates:
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: sriovnetwork.openshift.io/v1
+                        kind: SriovNetworkNodePolicy
+                        metadata:
+                            name: sriov-nnp-du-mh
+                            namespace: openshift-sriov-network-operator
+                        spec:
+                            deviceType: vfio-pci
+                            isRdma: false
+                            nicSelector:
+                                pfNames:
+                                    - ens7f0
+                            nodeSelector:
+                                node-role.kubernetes.io/worker: ""
+                            numVfs: 8
+                            resourceName: du_mh
+```
+Note: Define the required input parameters as $value (ex: $deviceType) is not mandatory. The $ tells the policy generator to remove this item from the generated policy if it does not given as input parameter. otherwise the value will stay as it is.
+
+3 - PolicyGenTemplate:
+
+PolicyGenTemplate is a Custom Resource Definition (CRD) that tells PolicyGen where to locate the generated policies and which Spec items need to be defined. Check the [PolicyGenTemplate](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/ztp-policy-generator/policyGenTemplates/policyGenTemplate.yaml) for more info. Let's consider the [group-du-ranGen.yaml](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/ztp-policy-generator/ranPolicyGenTemplateExamples/group-du-ranGen.yaml) example to explain more
+
+```
+apiVersion: policyGenerator/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du-policies"
+  namespace: "policy-template"
+  labels:
+    common: false
+    groupName: "group-du"
+    siteName: "N/A"
+    mcp: "worker-du"
+sourceFiles:
+  - fileName: MachineConfigPool
+    policyName: "mcp-du-policy"
+    name: "worker-du"
+  - fileName: SriovOperatorConfig
+    policyName: "sriov-operconfig-policy"
+    name: "N/A"
```

Comment:
Does the name field need to be specified, or will it default to N/A? If it defaults perhaps this can be cleaned up by removing those?

Reply:
will remove it